dask_sql/context.py

```diff
@@ -248,9 +253,28 @@ def create_table(
         if statistics:
             self.schema[schema_name].statistics[table_name.lower()] = statistics
 
+        dc_statistics = None
+        if parquet_statistics and not statistics:
+            try:
+                stats = parquet_statistics(input_table)
+                row_count = 0
+                if stats:
+                    # List of dicts
+                    for d in stats:
+                        row_count += d["num-rows"]
+                    dc_statistics = Statistics(row_count)
+            except ValueError:
+                dc_statistics = None
```

Comment:
So one thing about this PR is that it only works on Dask DataFrames that were read in with `read_parquet` and then added to the DaskSQLContext with `c.create_table("ddf", ddf)`. However, in our TPC-DS tests, we are skipping `read_parquet()` and just doing `c.create_table(table_name, my_filepath, format="parquet", gpu=True)`, so `parquet_statistics()` doesn't work on it. If we wanted to, we could do something like:

```suggestion
                if format.lower() == "parquet":
                    if dask_cudf:
                        ddf = dask_cudf.read_parquet(input_table)
                    else:
                        ddf = dd.read_parquet(input_table)
                    stats = parquet_statistics(ddf)
                    row_count = 0
                    if stats:
                        # List of dicts
                        for d in stats:
                            row_count += d["num-rows"]
                        dc_statistics = Statistics(row_count)
```

Does this seem reasonable? If I'm understanding it correctly, since we're not calling `ddf.compute()` it should be OK performance-wise?